src/doc/en/reference/documentation/conf.py

```diff
@@ -1 +1,55 @@
-../conf_sub.py
\ No newline at end of file
+../conf_sub.py
+
+
+def fast_callable(expression, vars=None, domain=None, hold=False, use_cache=False):
```

Comment:
That's not the right place to add it.

Look for the definition of `fast_callable`, for example, by typing `git grep 'def fast_callable' `. Change it there.